BuildResidentialHPXML/measure.rb

```diff
@@ -345,6 +345,13 @@ def arguments(model)
     arg.setDefaultValue(0.0)
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('geometry_rim_joist_height', true)
+    arg.setDisplayName('Geometry: Rim Joist Height')
+    arg.setUnits('ft')
```

Comment:
Let's make this units of inches so someone doesn't have to convert 7.25 inches (2x8) to feet.